extide/gradle/src/org/netbeans/modules/gradle/api/execute/GradleCommandLine.java

```diff
@@ -88,58 +89,82 @@
      * @since 2.23
      */
     public interface GradleOptionItem {
+        /**
+         * Shall return {@code true} if the IDE supports this option item.
+         *
+         * @return {@code true} if this option is supported by the IDE
+         */
         boolean isSupported();
+        /**
+         * Shall return {@code true} if this option is supported by the provided
+         * GradleVersion.
+         * 
+         * @param ver the GradleVersion to check.
+         * @return {@code true} if the provided Gradle version supports this option.
+         */
```

Comment:
`@since`

Reply:
There will be an updated commit with javadoc and tests and arch changes. Let's just check the code itself for now.